packages/sei-cosmwasm/README.md

```diff
@@ -57,6 +57,42 @@ Currently, Sei Bindings support query and message support for the sei custom mod
             - Change the Admin of the Denom. Only the current admin can change the admin.
         - SetMetadata
             - Set the denom metadata of a factory denom. Only the current admin can set metadata.
+- EVM
+    - Query
+        - StaticCall
+            - TODO
+        - Erc20TransferPayload
+            - Gets the Erc20 transfer payload from on recipient and amount
+        - Erc20TransferFromPayload
+            - Gets the Erc20 transfer from payload based on owner, recipient and amount
+        - Erc20ApprovePayload
+            - Gets the Erc20 approve payload from spender and amount
+        - Erc20Allowance
+            - Gets the Erc20 allowance from contract address, owner and spender
+        - Erc20TokenInfo
+            - Gets the Erc20 token info from contract address and caller
+        - Erc20Balance
+            - Gets the Erc20 balance from contract address and account
+        - Erc721TransferPayload
+            - Similar to the Erc20 equivalent
+        - Erc721ApprovePayload
+            - Similar to the Erc20 equivalent
+        - Erc721Approved
+            - Checks if a caller is approved based on caller, contract address and token id
```

Comment:
what operations is the caller approved for here?

Reply:
to send NFTs on behalf of the owner